charts/ceph-csi-rbd/README.md

```diff
@@ -111,6 +111,7 @@ charts and their default values.
 | `provisioner.minSnapshotsOnImage`              | Minimum number of snapshots allowed on rbd image to trigger flattening                                                                               | `250`                                              |
 | `provisioner.skipForceFlatten`                 | Skip image flattening if kernel support mapping of rbd images which has the deep-flatten feature                                                     | `false`                                            |
 | `provisioner.timeout`                          | GRPC timeout for waiting for creation or deletion of a volume                                                                                        | `60s`                                              |
+| `provisioner.clustername`                      | Cluster name to set on the RBD image                                                                                                                 | ""                                                 |
```

Comment:
is the default really `""`?

Reply:
Yes, this will let the old behavior persist.